gctscript/modules/gct/exchange.go

```diff
@@ -252,23 +268,24 @@ func ExchangeAccountInfo(args ...objects.Object) (objects.Object, error) {
 	data := make(map[string]objects.Object, 2)
 	data["exchange"] = &objects.String{Value: rtnValue.Exchange}
 	data["currencies"] = &funds
-
-	return &objects.Map{
-		Value: data,
-	}, nil
+	return &objects.Map{Value: data}, nil
 }
 
 // ExchangeOrderQuery query order on exchange
 func ExchangeOrderQuery(args ...objects.Object) (objects.Object, error) {
-	if len(args) < 2 {
+	if len(args) < 3 {
 		return nil, objects.ErrWrongNumArguments
 	}
 
-	exchangeName, ok := objects.ToString(args[0])
+	scriptCtx, ok := objects.ToInterface(args[0]).(*Context)
+	if !ok {
+		return nil, fmt.Errorf(ErrParameterConvertFailed, scriptCtx)
```

Comment:
Pretty minor, but this seems like a good time for `common.GetAssertError()` instead